### PR TITLE
+ tabs in header bar, moved sections (top, best, new)

### PR DESCRIPTION
**Pull request by ranfdev**
screenshot from 2019-03-05 22 42 51
As you can see from the image, i moved the ordering of the posts in the header bar to provide a more consistent experience with the rest of the system

### CODE REVIEW

File: src/MainWindow.vala [hunk 1]
```diff
@@ -40,11 +40,26 @@ public class MainWindow : Gtk.Window {
         set_position (Gtk.WindowPosition.CENTER);
 
         var header = new Gtk.HeaderBar ();
-        header.get_style_context ().add_class ("default-decoration");
-        header.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
```
Comment (mdh34) [inline src/MainWindow.vala]: Could we possibly keep this line for now? I'd like to do any style changes in a separate PR

**Comment by mdh34** [comment]
This looks great, just one little comment :)

**Comment by mdh34** [comment]
Looks great, thanks!

**Comment by ranfdev** [comment]
Whoops, i wrote a wrong commit message, this is not the PR with the comments. Nevermind :rofl:

**Comment by mdh34** [comment]
Oops, at least they're squashed
